opm/simulators/utils/MPIPacker.hpp

```diff
@@ -59,9 +67,14 @@ struct Packing<true,T>
     //! \param comm The communicator to use
     static std::size_t packSize(const T*, std::size_t n, Parallel::MPIComm comm)
     {
+        // For now we do not handle the situation where a a single call to packSize/pack/unpack
+        // is likely to require an MPI_Pack_size value larger than intmax
+        assert ( n*sizeof(T) <= std::numeric_limits<int>::max() );
         int size = 0;
         MPI_Pack_size(n, Dune::MPITraits<T>::getType(), comm, &size);
-        return size;
+        assert (size >= 0);
+        assert (size < std::numeric_limits<int>::max() );
```

Comment:
I think we can skip this assert as the type is an int.

Reply:
Certainly!